commit 511d5abcb5c79d03a4a0700fb4e6d5766c1148a4
Author: Thomas Bogendoerfer <tsbogend@alpha.franken.de>
Date:   Sat Aug 12 23:36:47 2017 +0200

    parisc: pci memory bar assignment fails with 64bit kernels on dino/cujo
    
    commit 4098116039911e8870d84c975e2ec22dab65a909 upstream.
    
    For 64bit kernels the lmmio_space_offset of the host bridge window
    isn't set correctly on systems with dino/cujo PCI host bridges.
    This leads to not assigned memory bars and failing drivers, which
    need to use these bars.
    
    Signed-off-by: Thomas Bogendoerfer <tsbogend@alpha.franken.de>
    Acked-by: Helge Deller <deller@gmx.de>
    Signed-off-by: Helge Deller <deller@gmx.de>
    Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>

Index: linuxtest/dummy/rpi_1634_511d5abcb5c79d03a4a0700fb4e6d5766c1148a4.txt
===================================================================
--- /dev/null
+++ linuxtest/dummy/rpi_1634_511d5abcb5c79d03a4a0700fb4e6d5766c1148a4.txt
@@ -0,0 +1 @@
+dummy file to ensure patch has content.
